docs/resources/compute_instance.md

```diff
@@ -66,6 +65,11 @@ EOF
 * `deploy_target_id` - A Deploy Target ID.
 * `labels` - A map of key/value labels.
 
+`network_interface` - Attach the compute instance to a private network (can be specified multiple times):
```

Comment:
```suggestion
`network_interface` - Attach the compute instance to a private network (can be specified multiple times):
```

I would also provide an actual usage example in the `resource "exoscale_compute_instance" "example"` stanza above (in this file)